monkey/infection_monkey/ransomware/file_selectors.py

```diff
@@ -19,7 +21,15 @@ def __call__(self, target_dir: Path) -> List[Path]:
             file_extension_filter(self._targeted_file_extensions),
             is_not_shortcut_filter,
             is_not_symlink_filter,
+            _is_not_ransomware_readme_filter,
         ]
 
         all_files = get_all_regular_files_in_directory(target_dir)
         return filter_files(all_files, file_filters)
+
+
+def _is_not_ransomware_readme_filter(filepath: Path) -> bool:
+    if filepath.name != README_FILE_NAME:
+        return True
+
+    return get_file_sha256_hash(filepath) != README_SHA256_HASH
```

Comment:
So it turns out that even on windows, the readme.txt dropped retains the unix line endings, good to know

Reply:
Yeah. I even opened the file in notepad, added a space, and saved it and the line endings were not changed. This will likely vary from editor to editor.